app/assess/routes.py

```diff
@@ -143,7 +147,7 @@ def score(
     )
     # TODO: Refactor this function so it doesn't rely on side-effects
     # and mutating SubCriteria model
-    determine_display_status(sub_criteria, flag)
+    determine_display_status(sub_criteria.workflow_status, flag)
```

Comment:
Do we need to do something with the return value here? And could we remove the TODO comment?

Reply:
Good spot fixed in 85ee0e9